lightningd/connect_control.c

```diff
@@ -140,7 +141,7 @@ static struct command_result *json_connect(struct command *cmd,
 		/* Is there a port? */
 		if (!port) {
 			port = tal(cmd, u32);
-			*port = DEFAULT_PORT;
+			*port = DEFAULT_PORT + chainparams->rpc_port - 8332;
```

Comment:
Why not add the default port to the `chainparams` instead of this derivation? It'd be clearer where the numbers come from.

Reply:
Yeah, I was thinking to move this derivation in some place, but we can have a static file in the chainparams.

Thanks!